scene/resources/navigation_mesh.cpp

```diff
@@ -337,6 +341,7 @@ void NavigationMesh::clear_polygons() {
 	polygons.clear();
 }
 
+#ifndef DISABLE_DEPRECATED
 Ref<Mesh> NavigationMesh::get_debug_mesh() {
 	if (debug_mesh.is_valid()) {
```

Comment:
Could maybe be completely removed? 4.0 breaks compat, and this doesn't seem exposed anyway?

Reply:
If I remember I had a reason for no removing this function immediately, like some other stuff in the engine was using the old debug mesh and function for the mesh data instead of reading the navmesh resource directly.